src/library/StatBoxList/Pie.tsx

```diff
@@ -30,7 +29,20 @@ export const Pie = ({ label, stat, graph, tooltip, helpKey }: PieProps) => {
     <StatBox>
       <div className="content chart">
         <div className="chart">
-          <StatPie value={graph?.value1} value2={graph?.value2} />
+          <Chart
+            items={[
+              {
+                value: graph?.value1,
+                color: 'var(--accent-color-primary)',
+              },
+              {
+                value: graph?.value2,
+                color: 'var(--background-default)',
+              },
+            ]}
+            diameter={30}
```

Comment:
Matching the size of current solution.
```suggestion
            diameter={34}
```